Sources/OktaIdx/Internal/Implementations/Version1/Responses/Responses.swift

```diff
@@ -57,7 +57,7 @@ extension IDXClient.APIVersion1 {
         let parameters: [String:Any]
     }
 
-    class Response: NSObject, Decodable {
+    final class Response: NSObject, Decodable {
```

Comment:
During researching IDX lib it was hard to understand whether `Response` is a base class and has the descendants. 
I can revert it :)